packages/web-config-server/package.json

```diff
@@ -5,15 +5,15 @@
   "description": "Config server for tupaia.org",
   "main": "dist",
   "scripts": {
-    "lint": "eslint --ext .js  --ignore-pattern __tests__/ .",
```

Comment:
So we used to ignore tests and now we include them. Just double checking that this is intentional?

Reply:
Yeap! Tests should be linted too (I think people excluded them in the past just for convenience 😆 )

We can still disable some rules in the tests folder (eg enforcing camelCase, since some times we use snake_case for test data), I'll do that if required in a next PR